pipeline.yaml

```diff
@@ -88,19 +89,6 @@ jobs:
     - template: templates/update_cli.yml
     - template: templates/conda.yml
     - template: templates/kv.yml
-    - bash: |
```

Comment:
We should keep this as-is, depending on publish takes too long (if everything is working correctly lol). The fix should be to publishLocal before python tests so that they can use local jars and not depend on cloud publishing

Reply:
Ah yes of course then, I see python tests already depend on publishLocal though.